.sphinx/_templates/footer.html

```diff
@@ -91,37 +91,28 @@
     {% endif %}
   </div>
   <div class="right-details">
-
-    {# mod: replaced RTD icons with our links #}
-
-    {% if discourse %}
-    <div class="ask-discourse">
-      <a class="muted-link" href="{{ discourse }}">Ask a question on Discourse</a>
-    </div>
-    {% endif %}
-
-    {% if mattermost %}
-    <div class="ask-mattermost">
-      <a class="muted-link" href="{{ mattermost }}">Ask a question on Mattermost</a>
-    </div>
-    {% endif %}
-
     {% if matrix %}
     <div class="ask-matrix">
       <a class="muted-link" href="{{ matrix }}">Ask a question on Matrix</a>
     </div>
     {% endif %}
 
+    {% if github_discussions %}
+    <div class="ask-github">
+      <a class="muted-link" href="{{ github_qa }}">Ask a question on GitHub</a>
+    </div>
+    {% endif %}
```

Comment:
Should this also set the title and description of the new discussion?

Reply:
On second thought, yes we should. It would be good for us to provide readers with guiderails when they jump into GitHub Discussions rather than just throw them into a random form page with no additional information.